### ISSUE TITLE
[OJS] Automatic Review Reminders should be "sent" by the assigned editor, not Principal Contact.

### ISSUE THREAD

**Issue by aukondk** [pkp/pkp-lib#6756]
**Describe the problem you would like to solve**
Automatic review reminders currently are "sent" by the Journal's Principal Contact. This is not ideal as it breaks the chain of communication between the editor and the reviewer.

**Describe the solution you'd like**
Ideally the reminder would be sent by the editor who requested the review but I think that is not expressly recorded.
Maybe just the top assigned editor of the review stage.

**Who is asking for this feature?**
Our Journal Editor is also Principal Contact but he is not always the editor dealing with reviews.

**Comment by ajnyga** [comment]
I agree, one of our journals just asked about this. The primary contact was confused when he was contacted regarding a submission he was not aware of.

**Comment by ajnyga** [comment]
see also:
https://forum.pkp.sfu.ca/t/automatic-email-reminders-sent-from-principal-contact-not-section-editor/73015

**Comment by asmecher** [comment]
We can't guarantee that there will be a single editor assigned to the workflow stage, and in other areas where we've attempted to "divine" the best candidate, the inconsistent behaviour has caused even more confusion. Thoughts are welcome on a consistent and sensible approach!

**Comment by ajnyga** [comment]
I think one approach could be that we make it more apparent that some of these messages are automatic and not sent by an individual.

**Comment by asmecher** [comment]
:+1: to that, @ajnyga -- I would support that. It would inform both the reviewer and the primary contact.

The only other approach I could think of -- and I don't love it -- would be to have the editorial assignment process include a designation for primary contact on a workflow stage, with defaults going e.g. to the first assignment. But that seems like working to further complicate the workflow and create additional confusion.
